src/versionbits.h

```diff
@@ -53,6 +61,7 @@ class AbstractThresholdConditionChecker {
 public:
     // Note that the functions below take a pindexPrev as input: they compute information for block B based on its parent.
```

Comment:
yeah, I'd suggest changing `GetStateSinceHeightFor()` in a different commit. I can open a PR for that or leave it to you if you prefer.